scripts/deploy-frontend.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+# source deploy-util.sh
+
+# The deployment strategy you wish to employ ( rolling update or setting up a new environment)
+DEPLOY_STRATEGY=${1}
+
+# The environment in which you want to execute these commands
+DEPLOY_ENV=${2}
+
+#The application name  defined via the manifest yml for the frontend
+CGHOSTNAME_FRONTEND=${3}
+CGHOSTNAME_BACKEND=${4}
+
+
+update_frontend()
+{
+
+
+    echo DEPLOY_STRATEGY: $DEPLOY_STRATEGY
+    echo DEPLOY_ENV: $DEPLOY_ENV
+    echo FRONTEND_HOST: $CGHOSTNAME_FRONTEND
+    echo BACKEND_HOST: $CGHOSTNAME_BACKEND
+    cd tdrs-frontend
+
```

Comment:
The docker file originally was responsible for populating the environment. We learned a while ago that we had to set the environment variables precompile, so had to have these commands ran in the docker file.

Now we need to have have them added locally, since the app is being built locally